rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -3345,7 +3478,12 @@ static void ppu_initialize2(jit_compiler& jit, const ppu_module& module_part, co
 	std::unique_ptr<Module> _module = std::make_unique<Module>(obj_name, jit.get_context());
 
 	// Initialize target
+#if defined(__APPLE__) && defined(ARCH_ARM64)
+	// Force target linux on macOS arm64 to bypass some 64-bit address space linking issues
+	_module->setTargetTriple(llvm::Triple::normalize("arm64-unknown-linux-gnu"));
+#else
 	_module->setTargetTriple(Triple::normalize(sys::getProcessTriple()));
```

Comment:
I think this one is overwritten somewhere later.

Reply:
I couldn't find anywhere where the module target gets overridden, plus it can't be overridden elsewhere or this force override hack wouldn't work.

Reply:
I think Nekotina was thinking of this: https://github.com/RPCS3/llvm-mirror/blob/509d31ad89676522f7121b3bb8688f7d29b7ee60/lib/Support/Triple.cpp

Which may be obsolete with your change?
(We did it because MachO backend has no trampoline generation for long jumps)

Reply:
Actually, this change is fine for now, sorry.